### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by marcelgerber**
Adds a custom 404 page to Brackets.

Screen (@larz0):

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div id="hero-wrapper" class="gradient">
+        <div id="hero" class="row no-bg-img">
+            <div class="large-12 columns">
+                <h1>404. That's an error.</h1>
+                <p>We can do many things, but we just can't show you an inexistent web page.</p>
```
Comment (njx) [inline 404.html]: Drive-by: "an inexistent" should be "a nonexistent"

File: 404.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                <p>Go check the URL or use one of the links in the nav bar above.</p>
```
Comment (njx) [inline 404.html]: Maybe "Please check the URL"? Also, we should probably use "navigation bar" or "navbar" ("nav bar" sounds weird to me).

**Comment by larz0** [comment]
@njx @MarcelGerber how about something shorter like:

# 404 because the page can't be found

Check the URL or go to the [Homepage](http://brackets.io).

**Comment by njx** [comment]
I kind of like the funny tone of the original (although I feel like I've seen the "404. That's an error." tagline somewhere else before, so we might want to come up with something different). But yeah, shorter would be good.

**Comment by larz0** [comment]
Maybe we can show different 404 messages randomly if @MarcelGerber also wants to have fun :)

**Comment by larz0** [comment]
"You found our 404 page!"

**Comment by peterflynn** [comment]
@njx Yeah, that string is from somewhere else -- I think Google uses it on all their sites?  Personally, I've never liked it though -- it's always come across a little supercilious to me... I think we could have a more friendly tone.

**Comment by njx** [comment]
Yup, that's kind of what I was thinking.

**Comment by peterflynn** [comment]
@larz0 Haha, I like that last suggestion

**Comment by marcelgerber** [comment]
"Dead Link. You may know that from Zelda."

**Comment by larz0** [comment]
1358422_9752001_lz

**Comment by larz0** [comment]
@adrocknaphobia would love that one.

**Comment by njx** [comment]
CONTENT WARNING PLZ

**Comment by larz0** [comment]
404 (WARNING: GRAPHIC)

**Comment by marcelgerber** [comment]
Guys, I can change the (any) text to anything you want, but I need to know what to change it to :cat: 

@peterflynn Yes, it's Google. See http://google.com/404

**Comment by larz0** [comment]
@MarcelGerber for now could you use:

# You found our 404 page!

Check if you have the right URL or go to the [Homepage](http://brackets.io).

**Comment by marcelgerber** [comment]
@larz0 Done.

**Comment by njx** [comment]
Sounds good and not at all traumatic.

**Comment by larz0** [comment]
Hehe cool. I'm going to merge this :+1:

**Comment by marcelgerber** [comment]
Works. Now the only thing missing is our custom 418 page (_I'm a teapot_) :laughing:

**Comment by njx** [comment]
@larz0 CONTENT WARNING AGAIN PLZ

**Comment by larz0** [comment]
cyclopscat

**Comment by lkcampbell** [comment]
The robot in the graphic in the Google 404 page looks like the Brackets self-help robot gone wrong:

http://blog.brackets.io/wp-content/uploads/2013/05/brackets-self-help-tee.png

**Comment by marcelgerber** [comment]
@lkcampbell We actually have that image on the server already: http://brackets.io/img/bg-brackey-bot.jpg
At first, I wanted to use it (probably inspired by the Google 404 page -- I took a look at some other 404 pages, of both well-known and rather unknown websites, to get inspired), but then I noticed it has a white background (instead of a transparent one) and therefore doesn't fit into a blue background.
